modules/python/python-dest.c

```diff
@@ -267,7 +267,15 @@ _py_free_bindings(PythonDestDriver *self)
 
 static gboolean
 _py_init_object(PythonDestDriver *self)
-{
+{   
+  if (!_py_get_attr_or_null(self->py.instance, "init"))
+    {
+      msg_error("Missing Python method, init()",
```

Comment:
The modification aims is to make the `init` function optional, therefore I think it is not appropriate to write error logs, if the `init` is missing. You could use `debug` level for this.

Reply:
done.